src/olympia/api/urls.py

```diff
@@ -1,11 +1,56 @@
+from django.conf import settings
 from django.urls import include, re_path
 
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
+
 from olympia.accounts.urls import accounts_v3, accounts_v4, auth_urls
 from olympia.addons.api_urls import addons_v3, addons_v4, addons_v5
 from olympia.amo.urls import api_patterns as amo_api_patterns
 from olympia.ratings.api_urls import ratings_v3, ratings_v4
 
 
+def get_versioned_api_routes(version, url_patterns):
+    route_pattern = r'^{}/'.format(version)
+
+    schema_view = get_schema_view(
+        openapi.Info(
+            title='AMO API',
+            default_version=version,
+            description='The official API for addons.mozilla.org.',
+        ),
+        public=True,
+        permission_classes=(permissions.AllowAny,),
+    )
+
+    routes = url_patterns
+
+    # For now, this feature is only enabled in dev mode
+    if settings.DEBUG:
```

Comment:
this means it will only be available locally, not even on addons-dev.allizom.org.  That's what you intended?

Reply:
I think it's okay like that for now. We can always change it. I believe it will be useful for prod, but we can figure out where it fits and change it then rather than creating a new setting.